src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>false</threadsafe>
```

Comment:
Why not?

Reply:
I don't know. This was the way it's done in our previously project.

Reply:
A simple web app should be thread safe if it has no mutable data structures.